NuGet.config

```diff
@@ -3,5 +3,7 @@
   <packageSources>
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v2" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-cli" value="https://www.myget.org/F/dotnet-cli/api/v3/index.json" />
```

Comment:
I may not be able to - I added `dotnet-core` to be able to get `"Microsoft.DotNet.ProjectModel": "1.0.0-*"`